packages/office-ui-fabric-react/src/components/Pivot/Pivot.scss

```diff
@@ -40,6 +40,13 @@
 
   &:hover {
     cursor: pointer;
+
+    @include high-contrast {
+      &::before {
+        box-sizing: border-box;
+        border-bottom: 2px solid Highlight;
```

Comment:
set this to be a permanent hover style outside of @include high-contrast, and then change the color for just IE